torch/utils/collect_env.py

```diff
@@ -203,6 +204,23 @@ def get_nvidia_smi():
     return smi
 
 
+def get_cpu_info(run_lambda):
+    rc = 0
+    out = ''
+    err = ''
+    if get_platform() == 'linux':
+        rc, out, err = run_lambda('lscpu')
```

Comment:
Why not `cat /proc/cpuinfo` or use cpuinfo provided by PyTorch?

Reply:
good point. I'm looking into this.

Reply:
`cpuinfo` dumps info of all individual cores. It is too detailed. What we need is aggregated info in high level provided by `lscpu`, like number of cores, how cores are distributed across numa nodes.